docs/guide/docker-builder-vm.md

```diff
@@ -64,8 +64,8 @@ docker build --cache-from myrepo/foo:latest \
 ### Dockerfile as a CI environment
 
 With Docker Builder there is no need to build and push custom containers so they can be used as an environment to run CI tasks in. 
-Cirrus CI can do it for you! Just specify path to a `Dockerfile` via `dockerfile` field for you container 
-declaration in `.cirrus.yml` like this:
+Cirrus CI can do it for you! Just specify path to a `Dockerfile` with the `dockerfile` field for you container 
+declaration for the `.cirrus.yml` like this:
```

Comment:
Did you mean "from the"?

Reply:
I see what you mean.  Let me try to reword that